frontend/public/components/sidebars/resource-sidebar.jsx

```diff
@@ -60,44 +65,35 @@ class ResourceSidebarWrapper extends React.Component {
   }
 }
 
-export const SampleYaml = ({ sample, loadSampleYaml, downloadSampleYaml }) => {
-  const { highlightText, header, subheader, img, details, templateName, kind } = sample;
-  return (
-    <li className="co-resource-sidebar-item">
-      <h5 className="co-resource-sidebar-item__header">
-        <span className="text-uppercase">{highlightText}</span> {header}{' '}
-        <span className="co-role-sidebar-subheader">{subheader}</span>
-      </h5>
-      {img && <img src={img} className="co-resource-sidebar-item__img" />}
-      <p className="co-resource-sidebar-item__details">{details}</p>
-      <button className="btn btn-link" onClick={() => loadSampleYaml(templateName, kind)}>
-        <PasteIcon className="co-icon-space-r" />
-        Try it
-      </button>
-      <button
-        className="btn btn-link pull-right"
-        onClick={() => downloadSampleYaml(templateName, kind)}
-      >
-        <DownloadIcon className="co-icon-space-r" />
-        Download YAML
-      </button>
-    </li>
-  );
-};
-
 const ResourceSchema = ({ kindObj }) => (
   <ExploreType kindObj={kindObj} scrollTop={sidebarScrollTop} />
 );
 
-export const ResourceSidebar = (props) => {
-  const { downloadSampleYaml, height, isCreateMode, kindObj, loadSampleYaml } = props;
+const ResourceSamples = ({ samples, kindObj, downloadSampleYaml, loadSampleYaml }) => (
+  <ResourceSidebarSamples
+    samples={samples}
+    kindObj={kindObj}
+    downloadSampleYaml={downloadSampleYaml}
+    loadSampleYaml={loadSampleYaml}
+  />
+);
+
+const ResourceSidebarContent = (props) => {
+  const {
+    downloadSampleYaml,
+    height,
+    isCreateMode,
+    kindObj,
+    loadSampleYaml,
+    yamlSampleObj,
```

Comment:
I'd give this a different name since it's not an object, but a list of objects

Reply:
Done.